astropy/timeseries/periodograms/lombscargle/implementations/fastchi2_impl.py

```diff
@@ -112,7 +114,9 @@ def lombscargle_fastchi2(
     # Now create an indexing scheme so we can quickly
     # build-up matrices at each frequency
     order = [("C", 0)] if fit_mean else []
-    order.extend(sum(([("S", i), ("C", i)] for i in range(1, nterms + 1)), []))
+    order.extend(
+        chain.from_iterable([("S", i), ("C", i)] for i in range(1, nterms + 1))
+    )
```

Comment:
We just have another PR by @eerovaher getting rid of `chain`... :+1: to the above suggestion - nobody has to think what `sum` does to a list or worry about chaining, just explicit what is done.

Reply:
If we don't like chain we can use the nested-list comprehension method. This should be faster than for-looping over an `extend`.

``[item for sublist in lists for item in sublist]``

Reply:
https://docs.astral.sh/ruff/rules/quadratic-list-summation/

Reply:
Using `chain(()` here is not a bad idea because the number of iterables being combined is variable. We should just avoid the lone `)` if doing so is as simple as it is here.

Reply:
Yes, it is possible, but I find @eerovaher's first suggestion much clearer - this is clearly a sin+cos series being created, with possibly a zero-order term.

Reply:
And the actual evaluation and summation of orders will definitely take far more time than their set-up.